tests/sklearn/test_sklearn_models.py

```diff
@@ -1605,31 +1605,11 @@ def test_predict_correctness(
 
 
 @pytest.mark.parametrize("model_class, parameters", MODELS_AND_DATASETS)
-@pytest.mark.parametrize(
```

Comment:
there's no point of simulating here (we are testing `encrypt` + `run` + `decrypt`)